let exceptionThrower = () => {
    throw new Error('Exception thrown')
}

module.exports = exceptionThrower